index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+      <img id="header-img" src="https://open-shelf.ca/wp-content/uploads/2015/03/twitter-logo-small-1024x576.jpg" alt="">
+      <nav id="nav-bar" >
+        <ul id="menu">
+          <a class="nav-link" href="#features"> features</a>
+          <a class="nav-link" href="#howitworks"> how it works</a>
+          <a class="nav-link" href="#pricing"> pricing</a>
+        </ul>
+      </nav>
+    </header>
+    <main id="main">
+      <h2>this is the home page of the store</h2>
+      <div class="">
```

Comment:
please do not use empty classes

Reply:
- [ ] Please try to  includes an explanation, a "why", for each requested change.
Example "Please Why don't you Apply best practices in your HTML code , leaving any empty classes in your HTML code makes your code-base not clear "